src/api.js

```diff
@@ -13,9 +13,11 @@ const getTrendingAlbums = () => {
 const getAlbumsByMasterId = (albumID) => {
   return fetch(`https://api.discogs.com/masters/${albumID}?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ`)
       .then(res => handleError(res))
-      // The two lines below are the additional endpoints for 'discovering albums', whoever has this ticket put these .then when you call this funtion and remove this comment
-      // .then(data => fetch(`https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&genre=${data.genres[0]}`))
-      // .then(res => handleError(res))
+}
+
+const getAlbumsByGenre = (genre) => {
+  return fetch(`https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=10&genre=${genre}`)
```

Comment:
Let's get the standard 50 that gets fetched and show a few random albums from that.

Reply:
done!